lib/src/scrollbar.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/gestures.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+/// Simulates the user interacting with a Scrollbar.
+extension ScrollbarInteractions on WidgetTester {
+  /// Press a scrollbar thumb at [thumbLocation] and drag it vertically by [delta] pixels.
+  Future<void> dragScrollbar(Offset thumbLocation, double delta) async {
```

Comment:
How does the user know where there scrollbar thumb is located?

Also, we should use this in a test in this package to demonstrate that it works, and show how we expect others to use it.